### PR TITLE
Fix Shortcuts help menu doesn't dismiss forever.

### LINKED ISSUE #1017: Shortcuts help menu doesn't dismiss forever
Steps to reproduce:
 - Go to Admin
 - See the shortcuts help menu in the bottom left
 - Click X to dismiss forever
 - Refresh page
 - Menu is back

### PR DESCRIPTION
**Pull request by Goyapa**
## What does this PR do?
Fix #1017

## How do I test this PR?
* Go to Admin
* See the shortcuts help menu in the bottom left
* Click X to dismiss forever
* Refresh page
* Menu has gone forever

### CODE REVIEW

**Comment by kgardnr** [comment]
Thank you @Goyapa! We will review this and send along any feedback for you. In the meantime, looking forward to seeing more from you for hacktoberfest! 🎉

**Review by kgardnr — APPROVED** [review]

**Comment by kgardnr** [comment]
This works for me! I'll let @okbel and @cvle approve the code before we merge. Thanks @Goyapa!

**Review by alinhle — CHANGES_REQUESTED** [review]
The root issue is here: 
https://github.com/coralproject/talk/blob/master/client/coral-framework/services/bootstrap.js#L182

You can see that `init` is not called. Can you change it there?

**Comment by Goyapa** [comment]
Thank you @cvle, I`ll have look!

**Comment by alinhle** [comment]
Thanks @Goyapa ! Remember init could be `undefined ` so maybe also default it to a noop.

**Comment by Goyapa** [comment]
@cvle could you please review once more. Hope it is now like you mentioned.

File: client/coral-framework/services/bootstrap.js [hunk 1]
```diff
@@ -178,6 +178,10 @@ export async function createContext({
     await preInit(context);
   }
 
+  if (init) {
+    await init(context);
+  }
+
   // Run initialization.
   await Promise.all([init, plugins.executeInit(context)]);
```
Comment (alinhle) [inline client/coral-framework/services/bootstrap.js]: It does work like you did. Like this it first runs and wait for `init` and then runs and wait for plugins init. It'd be more efficient to run both `init` and `plugins.executeInit` in one step and then wait for their promises to resolve.

E.g.

```js
import noop from 'lodash/noop';

export async function createContext({
   [...]
   init = noop,
 } = {}) {
  [...]
  await Promise.all([init(context), plugins.executeInit(context)]);
}
```

**Comment by Goyapa** [comment]
@cvle I see noop stands for lodash empty function. Next try! ;-)

**Review by alinhle — APPROVED** [review]
@Goyapa lgtm! merci

**Pull request merged**